README.md

```diff
@@ -405,21 +405,19 @@ await octokit.rest.issues.create({
 });
 ```
 
-The REST API endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID or parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
+The `octokit.rest` endpoint methods are generated automatically from [GitHub's OpenAPI specification](https://github.com/github/rest-api-description/). We track operation ID and parameter name changes in order to implement deprecation warnings and reduce the frequency of breaking changes.
 
-Every endpoint method is an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
+Under the covers, every endpoint method is actually an instance of `octokit.request` with defaults set, so it supports the same parameters as well as the `.endpoint()` API.
 
 #### Arbitrary requests
```

Comment:
Would be good to find a different name for this. This is really the `request` api but aside from a small number (hopefully zero) of APIs that have not made it into the schema, can I call more things than with `rest`? If so, what?

Reply:
It’s mostly used for 

1. id based routes, which are not yet part of the OpenAPI spec
2. Requests to other domains / other APIs
3. Some people prefer using the request method over the rest endpoint methods because there are fewer changes. Maintenance cost is lower over time

Reply:
1. Got it on the first one. 
2. Not sure I understand the second. What other domains (you mean GHES instances? Shouldn't that be handled automatically somehow?) Other APIs?
3. Completely see the style/aesthetic angle. May need to talk more about changes and maintenance. I would have thought it was the other way around (that `rest` is higher level and so would insulate people from underlying change).

Reply:
> 2. Not sure I understand the second. What other domains (you mean GHES instances? Shouldn't that be handled automatically somehow?) Other APIs?

entirely different APIs. If you build a GitHub app with Octokit but also need to talk to other API endpoints, then the easiest is to use `octokit.request("GET https://api.some-service.com/entities")`

> 3. Completely see the style/aesthetic angle. May need to talk more about changes and maintenance. I would have thought it was the other way around (that `rest` is higher level and so would insulate people from underlying change).

With the rest endpoint methods the `operationId` and the parameters `in: path` are significant. Any change basically results in a breaking change for the rest API. To avoid to many breaking changes, I keep track of them here and declare deprecations: https://github.com/octokit/openapi/tree/main/changes. The deprecations help with migrating to the new method names / parameter names, but just using `ocotkit.request` with a hard coded path prevents you from being bothered by all these changes in the first place